README.md

```diff
@@ -250,6 +236,66 @@ $ truckersmp-cli --wine --prefixdir "/path/to/prefix/pfx" start ets2mp
     * When using Proton, wined3d can be used by specifying `--use-wined3d`, but it's not recommended because this is slower than OpenGL.
 * Used only when `-d` or `--enable-d3d11` is specified.
 
+## Configuration file
+
+It's possible to set various game settings and third party programs in a configuration file. The default search path for the file is `$XDG_CONFIG_HOME/truckersmp-cli/truckersmp-cli.conf`.
+
+The configuration file format is similar to the INI format known from Windows with several sections `[section]` containing key-value-pairs `key = value`.
+All sections are optional and a key-value-pair has to be in a section.
+
+### Sections for game modes
+
+Supported sections are `[ats]`, `[astmp]`, `[ets2]` and `[ets2mp]`.
+#### Optional game mode settings
+
+Key-Value-Pair|Description
+---|---
+`without-rich-presence = true/false`|Disable discord rich presence for the game mode - overriding what is set in third party sections.
+
+### Sections for third party programs
+
+Third party programs in the configuration file will get started together with the game.
+Sections should follow the format `[thirdparty.gamemode.programname]` (one game mode) or `[thirdparty.programname]` (all game modes).
+
+#### Required third party settings
+
+Key-Value-Pair|Description
+---|---
+`executable = /path/to/my.exe`|Path to the executable where the path is either an absolute unix path (`/path/to/my.exe`), a relative unix path (`program/my.exe`) to `$XDG_DATA_HOME/truckersmp-cli/` or an absolute windows path (`C:\Program Files (x86)\program\my.exe`) within the prefix.
+
+#### Optional third party settings
+
+Key-Value-Pair|Description
+---|---
+`wait = seconds`|Set a minimal waiting time before the game will be started so third party programs have enough time to fully start up and log in.
+`wants-rich-presence = true/false`|Set that a program supports discords rich presence.
```

Comment:
> | will break the table

Can you use `,` instead of `/`? The format (`{value1,value2,...}`) is the same as `truckersmp-cli --help` output:
```
usage: truckersmp-cli [-h] [-a] [-b VERSION] [-d] [-e] [-g DIR] [-i APPID]
...
[{start,update,downgrade,updateandstart,ustart,downgradeandstart,dstart,none}]
[{ets2mp,ets2,atsmp,ats,none}]
```

Reply:
`getboolean()` in #219 can handle both `{yes,no}` and `{true,false}` (`{1,0}` is ok too): Users can specify `wants-rich-presence = yes` instead of `wants-rich-presence = true`. Do you prefer to use `{yes,no}` in the document?

Reply:
I don't know, I just used true,false because I'm used to it.